src/components/common/applicant-info-1.vue

```diff
@@ -483,10 +486,10 @@ export default class ApplicantInfo1 extends Mixins(ActionMixin) {
 
   get jurisdictionOptions () {
     return (this.getLocation === Location.CA)
-      ? this.$canJurisdictions.filter(jur => jur.value !== Location.BC)
-        .map(jurisdiction => ({ value: jurisdiction.text, text: jurisdiction.text }))
-      : this.$intlJurisdictions.filter(jur => jur.value !== Location.CA)
-        .map(jurisdiction => ({ value: jurisdiction.text, text: jurisdiction.text }))
+      ? CanJurisdictions.filter(jur => jur.value !== Location.BC)
+        .map(jurisdiction => ({ value: jurisdiction.text.toUpperCase(), text: jurisdiction.text }))
```

Comment:
What's the purpose of this? It will capitalize all keys in the v-select, right? Why have text in Title Case in the first place?

Reply:
This will only capitalize the value, as the display text is not full caps. (ie `value: ALBERTA, text: Alberta`)
This is because from Name Examination the Jurisdiction key value is capitalized.

The other option is changed the jurisdiction value properties to all capitalized just not sure of all the implications outside of this context.

Reply:
Gotcha.

But the thing that bugs me here is that it's unclear which code needs what or what value is at any one time. I'd be tempted to add a property to every juridiction object named `namexValue` that is the upper case value. It's explicit and more clear IMO. What do you think?

Reply:
I like the idea actually! 
Although i might prefer to change the `value` to be capitalized so it's in sync with Name Ex, followed by exploring/fixing any implications of that, is that preferable vs a new property? 
Only concern was we would have the same value 3 times with just varying capitalization.

Reply:
Yes, just making it all uppercase is preferable... except it will change the v-select items, won't it?

It's not worth a ton of time deliberating over this. Less duplication is usually better, except when it makes things harder to understand. Your call.

Reply:
Ah i see what you mean by an additional property now. 
Yea, changing this value would reformat the text in the list as well :/ 
I think i prefer converting to uppercase to match the model vs additional properties but i can see it is a little confusing so i added a comment!

Reply:
I'm OK with that.